src/Command/Up.php

```diff
@@ -28,8 +29,10 @@ public function configure()
     {
         $this
             ->setName('up')
+            ->setAliases(['start'])
```

Comment:
I like it. So we're not doing assumed file sync any more?

Reply:
Exactly, file sync will still exist but you'll need to manually invoke it... Handy for vendor edits etc